cmd/botkube/main.go

```diff
@@ -195,8 +223,51 @@ func newMetricsServer(log logrus.FieldLogger, metricsPort string) *httpsrv.Serve
 	return httpsrv.New(log, addr, router)
 }
 
+func newAnalyticsReporter(disableAnalytics bool, logger logrus.FieldLogger) (analytics.Reporter, func(), error) {
+	if disableAnalytics {
+		logger.Info("Analytics disabled. Using noop reporter...")
```

Comment:
```suggestion
		logger.Info("Analytics disabled via configuration settings.")
```
IMO that enough, `noop` is more about the impl.